src/System.IO.Compression/src/Resources/Strings.resx

```diff
@@ -154,7 +154,7 @@
     <value>Stream does not support writing.</value>
   </data>
   <data name="ObjectDisposed_StreamClosed" xml:space="preserve">
-    <value>Can not access a closed Stream.</value>
+    <value>Cannot access a closed Stream.</value>
```

Comment:
Lowercase Stream (and above) as it's not starting a sentence.

Reply:
done